tests/test_util.py

```diff
@@ -96,3 +96,14 @@ def test_proportions_from_distribution():
     uniform = u.column(1)
     assert len(uniform) == 50 and _round_eq(1, sum(uniform))
     assert [x in (0, 0.5, 1) for x in ds.sample_proportions(2, ds.make_array(.2, .3, .5))]
+
+
+def test_is_non_string_iterable():
+    is_string = 'hello'
+    assert ds.is_non_string_iterable(is_string) == False
+
+    is_list = [1, 2, 3]
+    assert ds.is_non_string_iterable(is_list) == True
+
+    is_int = 1
+    assert ds.is_non_string_iterable(is_int) == False
```

Comment:
Can we add the remaining case that's not covered by tests right now: https://coveralls.io/builds/56773172/source?filename=datascience%2Futil.py#L257

Reply:
Thanks for the feedback @adnanhemani, I've removed the import and updated with a unit test for the Sequence type.